e2e/tests/omnibus-sepolia.toml/LegacyMarket.e2e.js

```diff
@@ -0,0 +1,117 @@
+const assert = require('assert');
+const { migrateLegacyMarket } = require('../../tasks/migrateLegacyMarket');
+const { getLpPosition } = require('../../tasks/getLpPosition');
+const { getAccountCollateral } = require('../../tasks/getAccountCollateral');
+const { ethers } = require('ethers');
+
+const log = require('debug')(`e2e:${require('path').basename(__filename, '.js')}`);
+
+const run = function () {
+  let LegacyMarketProxy;
+  let CoreProxy;
+  let V2xSnx;
+
+  // some big wallets
+  // https://etherscan.io/token/0x89FCb32F29e509cc42d0C8b6f058C993013A843F#balances
+  const accounts = [
+    '0x99F4176EE457afedFfCB1839c7aB7A030a5e4A92',
+    '0x8cA24021E3Ee3B5c241BBfcee0712554D7Dc38a1',
+    '0x27Cc4d6bc95b55a3a981BF1F1c7261CDa7bB0931',
+    '0xcb68110C43C97b6051FEd5e2Bacc2814aDaD1688',
+    '0x722f0a6F970F8B144F5e92aB3f3BeaFA280b282f',
```

Comment:
these are empty account on sepolia

Reply:
yea I know. we need to put actual addresses for sepolia but i'm not sure of one

Reply:
so focusing on testing mainnet